2012-08-29 22:16:10
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/proc.py", line 88, in run
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 03:48:59
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/proc.py", line 88, in run
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:04:52
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:35:21
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:35:53
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:35:57
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:36:02
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:36:18
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:36:27
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 04:59:38
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-08-30 05:00:04
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 15:58:53
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 15:58:53
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 15:59:03
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:07:02
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:07:07
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:07:18
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:08:37
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:11:38
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:11:47
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:12:41
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:13:07
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-09-05 16:15:18
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
